src/dtw.jl

```diff
@@ -49,6 +56,59 @@ function dtw_bhattacharyya(seq1::Array{kmerdist}, seq2::Array{kmerdist})
     return trackback_bhattacharyya(D)
 end
 
+function dtw_multiple_cost_matrix_bhattacharyya(seq::Array{Array{Float32}},
+    segmentsize = 100, transportcost=1) where T
+
+    #split the data into segments of size segmentsize
+    #note: if the sequences doesn't cleanly split into
+    #segments, it will just discard the remaining data
```

Comment:
1. Not sure what it is suposed to achieve, we already have kmerdist grouping functions why not use them?
2. Discarding data is NOT ok.